### PR TITLE
Bump to 1.310

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge-admin, please rerender

**Comment by hadim** [comment]
@conda-forge-admin, please rerender

**Review by tsenapathi — APPROVED** [review]

**Comment by tsenapathi** [comment]
Thanks @hadim for the update @moorepants do you have any comments?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,11 +22,10 @@ requirements:
     - python
     - pip
   run:
-    - python
+    - python >=3.6
```
Comment (moorepants) [inline recipe/meta.yaml:25]: Should this pin be in host and run?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -22,11 +22,10 @@ requirements:
...
     - numpy
     - xmltodict
     - beautifulsoup4
-    - matplotlib-base
     - requests
```
Comment (moorepants) [inline recipe/meta.yaml]: When I checked the new repo, I'm only seeing requests imported in the two files. Can all of these other dependencies be removed?

Comment (moorepants) [reply recipe/meta.yaml]: The conda meta.yaml in the main repo still has all of these dependencies: https://github.com/williamgilpin/pypdb/blob/master/devtools/conda-recipe/meta.yaml but it isn't clear to me why.

Comment (moorepants) [reply recipe/meta.yaml]: See that requests is all that is imported here:

https://github.com/williamgilpin/pypdb/blob/master/pypdb/pypdb.py#L24

Comment (moorepants) [reply recipe/meta.yaml]: Requests is the only thing requred in setup.py too: https://github.com/williamgilpin/pypdb/blob/master/setup.py#L9

Comment (hadim) [reply recipe/meta.yaml]: It looks like a lot of code has been commented on. Probably due to the soon-to-be-coming refactoring. I guess we should remove those deps for the moment.

**Comment by hadim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 2 above)
Comment (hadim) [reply recipe/meta.yaml]: beautifulsoup4 and xmltodict are still required in 1.310 (check the tag on github).

**Comment by hadim** [comment]
LGTM

**Review by tsenapathi — APPROVED** [review]

**Pull request merged**